pkg/gadgets/profile/block-io/tracer/bpf/biolatency.bpf.c

```diff
@@ -15,11 +15,10 @@ const volatile bool targ_per_disk = false;
 const volatile bool targ_per_flag = false;
 const volatile bool targ_queued = false;
 const volatile bool targ_ms = false;
+const volatile bool targ_single = true;
```

Comment:
There are 2 tracepoints:
* btf_trace_block_rq_insert
* btf_trace_block_rq_issue

But you test only one of them. In practice it works because https://github.com/torvalds/linux/commit/a54895fa057c67700270777f7661d8d3c7fda88a changed both at the same time. But it would be cleaner to test both, define two `const volatile` for each tracepoint,and use `RewriteConstants` on both.

Or at least add a comment before `targ_single` to explain its semantics and list the tracepoints.